autobuild/osx/app-template/Contents/Info.plist

```diff
@@ -4,9 +4,11 @@
 <dict>
 	<key>CFBundleDevelopmentRegion</key>
 	<string>English</string>
+	<key>CFBundleIdentifier</key>
+	<string>$(org.synfig.SynfigStudio)</string>
```

Comment:
$(org.synfig.SynfigStudio)
should be without $(), like this
```
<string>org.synfig.sifz</string>
```

Reply:
@ice0 Can you tell me how to rectify this without increasing number of commits :(

Thanks :)